Makefile

```diff
@@ -25,15 +25,21 @@ build:
 
 install:
 	rbenv install -s
-	- gem install bundler -v 2.2.33
+	- gem install bundler -v 2.2.33 && rbenv rehash
 	bundle install --jobs 1
 	cp -n config/connectors.yml.example config/connectors.yml || true
 
 run:
 	${YQ} e ".revision = \"$(shell git rev-parse HEAD)\"" -i config/connectors.yml
 	${YQ} e ".repository = \"$(shell git config --get remote.origin.url)\"" -i config/connectors.yml
 	${YQ} e ".version = \"$(shell script/version.sh)\"" -i config/connectors.yml
-	cd lib/app; bundle exec rackup config.ru
+	cd lib/app; bundle exec rackup --host 0.0.0.0 config.ru
+
+build-docker:
+	docker build -t connectors .
+
+run-docker:
```

Comment:
Does it depend on `build-docker`? If so, should we also declare the dependency and build the image if it's not built?

Reply:
yes and no. if you run it without the image built, it'll fail. But it should run without re-buidling everytime. I can't think of a simple mechanism to trigger the build if it was not done when running

Reply:
Fair point, I was also on the fence whether it should be a hard requirement. So if the docker image was never built, we still get a somewhat meaningful error message, right?

Reply:
```
% docker run --rm -it -p 127.0.0.1:9292:9292/tcp connectors
Unable to find image 'connectors:latest' locally
docker: Error response from daemon: pull access denied for connectors, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```